### ISSUE TITLE
JAWS not announcing item/position count when traversing through native HTML unordered lists

### ISSUE THREAD

**Issue by anuvenkatesh1** [FreedomScientific/standards-support#764]
As we navigate through a native unordered HTML list of buttons/links etc, why is JAWS not announcing the position count of the focused list item? Is this a known JAWS behavior with native HTML lists?
Does JAWS plan to address this?

**Expected result**
Expecting JAWS to announce the list item's position count eg: Google - 1 of 3, Yahoo - 2 of 3 etc. However this didn't get announced either in forms or in VPC mode.
While WCAG recommends to use native HTML as much as possible as too much ARIA could result in conflicting behavior with different ATs, due the lack of support for this item/position count announcement, we are forced to use list/list item/aria-setsize/aria-posinset etc. It would look a lot cleaner and simpler if we are able to use HTML lists and still have JAWS announce the position count.  

**Actual result**
![image](https://github.com/FreedomScientific/standards-support/assets/32471459/4ac1f9fe-ce1e-4897-bbec-8c1d69fd002b)

**JAWS version and build number**
Same behavior with all versions of JAWS until JAWS 2023

**Operating System and version**
Windows 10

**Browser and version:**
Chrome latest.

**Comment by JAWS-test** [comment]
I would not call this a bug of JAWS. 
It is the decision of JAWS which information is output. It could be a feature request that there is an option in the settings that the position of the list entries can also be perceived in lists.

Note: The position of the list entries is not output by NVDA either.

> we are forced to use list/list item/aria-setsize/aria-posinset
 
No, because with these ARIA roles and attributes the output is not done either. This is actually a JAWS bug that also occurs with NVDA.

**Comment by stevefaulkner** [comment]
closing this as wontfix as per @JAWS-test comment.
